packages/router/addon/index.js

```diff
@@ -22,22 +21,22 @@ if (macroCondition(getGlobalConfig()['@embroider/core']?.active)) {
     // mappings from routeName to the engines "original name" (which we know at build time).
     let engine = engineInfoByRoute[routeName];
     if (engine && window._embroiderEngineBundles_) {
-      return window._embroiderEngineBundles_.find(bundle => bundle.names.indexOf(engine.name) !== -1);
+      return window._embroiderEngineBundles_.find(
+        (bundle) => bundle.names.indexOf(engine.name) !== -1
+      );
     }
 
     if (window._embroiderRouteBundles_) {
-      return window._embroiderRouteBundles_.find(bundle => bundle.names.indexOf(routeName) !== -1);
+      return window._embroiderRouteBundles_.find(
+        (bundle) => bundle.names.indexOf(routeName) !== -1
+      );
     }
 
     return false;
   };
 
+  // eslint-disable-next-line ember/no-classic-classes
```

Comment:
I kept the classic class, because without it the provide `this._routerMicrolib` API was no longer present, and I would like to (if we decide to) do that upgrade independent of this one.